CMakeLists.txt

```diff
@@ -36,20 +36,20 @@ cmake_policy(SET CMP0003 NEW)
 #      cmake_policy(SET CMP0020 OLD)
 #endif(POLICY CMP0020)
 
-# Issue no warning non-existent target argument to get_targer_property() and set the result variable to a -NOTFOUND value rather than issuing a FATAL_ERROR
-if(POLICY CMP0045)
-      cmake_policy(SET CMP0045 OLD)
-endif(POLICY CMP0045)
+# Issue no warning non-existent target argument to get_target_property() and set the result variable to a -NOTFOUND value rather than issuing a FATAL_ERROR
+#if(POLICY CMP0045)
```

Comment:
Let's remove it instead of commenting it out. Git keeps history if needed.

Reply:
True. Will do shortly.